schema/crdb/make-link-settings-non-nullable/up1.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE omicron.public.switch_port_settings_link_config ALTER COLUMN speed SET NOT NULL;
+ALTER TABLE omicron.public.switch_port_settings_link_config ALTER COLUMN mtu SET NOT NULL;
+ALTER TABLE omicron.public.switch_port_settings_link_config ALTER COLUMN port_settings_id SET NOT NULL;
```

Comment:
This looks a *lot* like https://github.com/oxidecomputer/omicron/tree/main/schema/crdb#211-adding-a-new-column-without-a-default-value -- if there are any databases which exist, where any of these values **are** NULL, then we'd fail to apply the schema upgrade and the rack would be stuck.

1. Are we confident that these are non-NULL across **all** deployments? Why?
2. Could we use a default value here, in the case that there **are** NULL values?